hera_pspec/pspecdata.py

```diff
@@ -1172,11 +1305,14 @@ def get_G(self, key1, key2, exact_norm=False, pol=False):
         if (exact_norm):
             integral_beam = self.get_integral_beam(pol)
             del_tau = np.median(np.diff(self.delays()))*1e-9
+        if exact_norm:
+            qnorm =  del_tau * integral_beam
+        else:
+            qnorm = 1.
         for ch in range(self.spw_Ndlys):
-            if exact_norm: Q1 = self.get_Q_alt(ch) * del_tau * integral_beam
-            else: Q1 = self.get_Q_alt(ch)
-            Q2 = Q1
-            iR1Q1[ch] = np.dot(R1, Q1) # R_1 Q
+            Q1 = self.get_Q_alt(ch) * qnorm
+            Q2 = self.get_Q_alt(ch, include_extension=True) * qnorm
+            iR1Q1[ch] = np.dot(np.conj(R1).T, Q1) # R_1 Q
```

Comment:
I've added the following comment to both get_H and get_H 

            #G is given by [E^\alpha C,\beta]
            #where E^\alpha = R_1^\dagger Q^\apha R_2
            #C,\beta = Q2 and Q^\alpha = Q1
            #Note that we conjugate transpose R
            #because we want to E^\alpha to
            #give the absolute value squared of z = m_\alpha R x
            #where F is the fourier transform.
            #Q is essentially m_\alpha^\dagger m
            # so we need to sandwhich it between R_1^\dagger and R_2